packages/core/src/compile.ts

```diff
@@ -65,8 +66,8 @@ function extractParamTypes(
     } else if (isDateElement(element) || isTimeElement(element)) {
       params[element.value] = 'Date | number';
     } else if (isTagElement(element)) {
-      params[element.value] = '(v: ReactNode) => ReactNode';
-      imports.add(`import { ReactNode } from 'react';`);
+      params[element.value] = 'FormatXMLElementFn<T>';
+      imports.add(`import { FormatXMLElementFn } from 'intl-messageformat';`);
```

Comment:
Should we be re-exporting this? The consumer doesn't have a direct dep on it yeah?